RELEASE/scripts/sl_ascend.ash

```diff
@@ -12563,6 +12563,12 @@ boolean L9_oilPeak()
 			print("Checking to see if we should do the tavern while we are running high ML.", "green");
 			set_property("sl_forceTavern", true);
 
+			// Remove Driving Wastefully if we had it
```

Comment:
Why bother? It's not like it stops us from getting another buff when we actually want to get one, and it's harmless.

Reply:
Somehow it does, possibly the Asdon Bull function is broken? If I don't put this in it keeps driving Wastefully when it goes to the the next area. I found this while trying to figure out why it wasn't buffing Tavern properly.